hubmap_api_py_client/external.py

```diff
@@ -163,12 +163,83 @@ def _add_method(output_type, ResultsSetSubclass, doc):
 
 
 for output_type, doc in {
-    'cell': 'TODO: document cell',
-    'organ': 'TODO: document organ',
-    'gene': 'TODO: document gene',
-    'cluster': 'TODO: document cluster',
-    'dataset': 'TODO: document dataset',
-    'protein': 'TODO: document protein'
+    'cell':
+    '''
+    Select a set of cells. If no params are provided, selects the set of all cells.
+    Otherwise, selects a set of cells filtered based on parameters supplied.
+        where->str The type of entity being supplied as input to query, must be one of
+            ["cell", "gene", "protein", "organ", "dataset"]
+        has->List[str] A list of entities or expressions supplied as input to the query
+        genomic_modality->str Modality to consider in quantitative queries.
+            Required for queries in which "where" is "gene". Must be one of ["rna", "atac"].
+        logical_operator->str The logical operator applied to filters generated by elements of has.
+            Required for queries in which "where" is "gene" or "protein" and
+            "has" contains more than one element. Must be one of ["and", "or"].
+    ''',
+    'organ':
+    '''
+    Select a set of organs. If no params are provided, selects the set of all organs.
+    Otherwise, selects a set of organs filtered based on parameters supplied.
+        where->str The type of entity being supplied as input to query. Must be one of
+            ["gene", "organ"].
+        has->List[str] A list of entities supplied as input to the query
```

Comment:
more precise, provide example?

Reply:
Sure, I wasn't sure how much we wanted to get into examples in the doc string, but I can include some.

Reply:
Thanks: I think this is just another case where "entities" doesn't tell me a lot: I don't think we need to quote actual IDs, but we do need to tell users what information they already have can be used here.